application/src/main/java/org/opentripplanner/street/model/vertex/OsmElevatorVertex.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.street.model.vertex;
+
+import org.opentripplanner.framework.geometry.WgsCoordinate;
+import org.opentripplanner.osm.model.OsmEntityType;
```

Comment:
I'm currently removing references from the street model to other models and this introduces a new one. This type needs to be inside the street model or you need to come up with another solution to break the relationship.